.github/ISSUE_TEMPLATE.MD

```diff
@@ -0,0 +1,15 @@
+# Description
+
+# Steps to reproduce
+1.
+2.
+3.
+
+# Expected result
+
+# Screenshots
+
+# Logged in as
+
+# Environment
+Development || Test || Pre Production || Production
```

Comment:
I've always been partial to the environment definitions: `Development | Staging | Production`
Can we use those?

At the very least I don't see us using 4 environments any time soon